Contract/Election.sol

```diff
@@ -37,7 +37,7 @@ contract SampleVoting {
 
     function addCandidate (string memory _name) private {
         candidates[candidatesCount] = Candidate(candidatesCount, _name, 0);
-        candidatesCount ++;
```

Comment:
you changed the space ?

Reply:
yeah.

Reply:
Ok will give you points....but in real projects or open-source don't request a pull for one space

Reply:
Oh. okay !